tidy3d/plugins/adjoint/components/simulation.py

```diff
@@ -433,6 +435,123 @@ def to_simulation(self) -> Tuple[Simulation, JaxInfo]:
 
         return sim, jax_info
 
+    def to_gds(
+        self,
+        cell,
+        x: float = None,
+        y: float = None,
+        z: float = None,
+        permittivity_threshold: pd.NonNegativeFloat = 1,
+        frequency: pd.PositiveFloat = 0,
+        gds_layer_dtype_map: Dict[
+            AbstractMedium, Tuple[pd.NonNegativeInt, pd.NonNegativeInt]
+        ] = None,
+    ) -> None:
+        """Append the simulation structures to a .gds cell.
+
+        Parameters
+        ----------
+        cell : ``gdstk.Cell`` or ``gdspy.Cell``
+            Cell object to which the generated polygons are added.
+        x : float = None
+            Position of plane in x direction, only one of x,y,z can be specified to define plane.
+        y : float = None
+            Position of plane in y direction, only one of x,y,z can be specified to define plane.
+        z : float = None
+            Position of plane in z direction, only one of x,y,z can be specified to define plane.
+        permittivity_threshold : float = 1.001
```

Comment:
weird that it's 1.001 in the docstring but 1 in the default, do we know what's up with that?

Reply:
Huh interesting, I missed that. But it has been around at least since eea873a. I'll change the docstring.